client/src/pages/OrganizationProfile.js

```diff
@@ -185,6 +191,45 @@ const OrganizationProfile = () => {
   };
 
   const renderProfileData = () => {
+
+    const postDelete = async (post) => {
```

Comment:
Why `postDelete` and `handleEditPost` functions are inside `renderProfileData` function?

Reply:
good point, moved them to `organizationProfile`.